docs(contributing): basic fixes (#1418)
docarray/docarray: CONTRIBUTING.md

Bonus: **Know when to break the rules**. Documentation writing is as much art as it is science. Sometimes you will have to deviate from these rules in order to write good documentation.

#### [MyST](https://myst-parser.readthedocs.io/en/latest/) Elements Usage

1. Use the `{tab}` element to show multiple ways of doing one thing. [Example](https://docarray.jina.ai/fundamentals/document/#document) 
2. Use the `{admonition}` boxes with care.
3. Use `{dropdown}` to hide optional content, such as long code snippets or console output.

> Note: Example code in the documentation should also follow our [code style conventions]().

Reviewer: you u remove this section. it is not relevant anymore as we don't use myst

Author: Roger that. What about the `[code style conventions]()`? There's no URL

Reviewer: you can remove the link, lets just say follow our code style

> bash makedoc.sh local-only

Reviewer: ```suggestion
bash makedoc.sh
```

```

> The docs website will be generated in `_build/dirhtml`.

Reviewer: ```suggestion
The docs website will be generated in `site`.
```

To serve it, run:

```bash

> cd _build/dirhtml

Reviewer: ```suggestion
mkdocs serve
```
